layers/meta-resin-allwinner/conf/machine/bananapi-m1-plus.conf

```diff
@@ -8,3 +8,4 @@ include conf/machine/bananapi.conf
 
 KERNEL_DEVICETREE = "sun7i-a20-bananapi-m1-plus.dtb"
 UBOOT_MACHINE = "bananapi_m1_plus_config"
+PREFERRED_VERSION_linux-mainline = "4.14.15"
```

Comment:
Why was this change done? And why is it in this PR?

Reply:
BananaPi was tested with the 4.14 linux-mainline kernel. After adding the 4.15 linux-mainline for OrangeP, yocto started building 4.15 kernel for BananaPi also, probably because Yocto pulls the newest version of a recipe unless specified otherwise. This change enforces BananaPi machine to build the kernel it was tested on.